onnx/hub.py

```diff
@@ -363,12 +363,35 @@ def download_model_with_test_data(
                 "download the model from the model hub."
             )
 
+    def members_filter(tarfile, base):
+        result = []
+        for member in tarfile:
+            member_path = os.path.join(base, member.name)
+            abs_base = os.path.abspath(base)
+            abs_member = os.path.abspath(member_path)
+            if not abs_member.startswith(abs_base):
+                raise AssertionError(
+                    "The tarball file in downloading model contains directory traversal sequence which may contain harmful payload."
```

Comment:
Possible to print both paths in the error message for clarity?

Reply:
Added member path in error message